obal/data/roles/fetch_sources/tasks/main.yml

```diff
@@ -112,22 +117,12 @@
         - '%global _release'
         - '%global release'
 
-    - name: 'add changelog entry'
-      lineinfile:
-        path: "{{ spec_file_path }}"
-        line: |
-          * {{ lookup('pipe', 'date +"%a %b %d %Y"') }} {{ downstream_changelog_name | default('Satellite6 Jenkins') }} {{ downstream_version }}-{{ downstream_release }}
-          {{ downstream_changelog }}
-        insertafter: '^%changelog'
+    - name: 'ensure changelog'
+      changelog:
+        spec: "{{ spec_file_path }}"
+        changelog: "{{ changelog }}"
+      when: changelog is defined
 
     - name: 'update source files'
       include_role:
         name: setup_sources
-
```

Comment:
this means we *always* add a changelog entry, which I think we should *not* do in the case of a downstream "copy from upstream" scenario.

Reply:
I think this is addressed by -- https://github.com/theforeman/obal/pull/68#discussion_r215586801

Reply:
yes it is